pkg/bootstrap/instance_test.go

```diff
@@ -442,12 +457,14 @@ func checkListStringMatcher(t *testing.T, got *matcher.ListStringMatcher, want s
 		}
 
 		if pat != "" {
-			patterns = append(patterns, pat)
+			gotPatterns = append(gotPatterns, pat)
 		}
 	}
-	gotPattern := strings.Join(patterns, ",")
-	if want != gotPattern {
-		t.Fatalf("%s mismatch:\ngot: %s\nwant: %s", typ, gotPattern, want)
+
+	sort.Strings(want)
```

Comment:
why is sort needed? DeepEqual doesn't care about ordering afaik

Reply:
I think it does for a list

Reply:
ok.. thanks!